Sources/performance-worker/RPCStats.swift

```diff
@@ -129,4 +129,16 @@ struct RPCStats {
       }
     }
   }
+
+  @available(macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0, *)
+  mutating func update(withStatsFrom clients: [BenchmarkClient]) throws {
```

Comment:
This unnecessarily couples the `BenchmarkClient` to `RPCStats`; the only way to update it is using a `BenchmarkClient` which is unnecessary and makes testing it more difficult.

I think the API you want here is the simplest one: `merge(_ other: RPCStats)`, the caller can then iterate over benchmark clients and call this function each time.